Core1Platform/AdministratingCore1/03_ManageGroups.md

```diff
@@ -0,0 +1,186 @@
+
+## Create user group
+
+Create a user group based on a user role to subsequently assign them a specific set of user rights.
+
+#### Prerequisites
+
+- You are granted to edit, save, and list groups.
+
+#### Procedure
+
+*Settings > Users and groups > Tab GRUPPENVERWALTUNG (GROUP MANAGEMENT)*
+
+![Group management](../../Assets/Screenshots/Core1Platform/AdministratingCore1/GroupCreated.png "[Group management]")
+
+1. Click the ![Add](../../Assets/Icons/Plus01.png "[Add]") (Add) button in the bottom right corner.    
+    The *Create group* window is displayed.
+
+    ![Create group](../../Assets/Screenshots/Core1Platform/AdministratingCore1/GroupCreate.png "[Create group]")
+
+2. Enter a name for the group in the *Group name* field.
+
+3. Click the [SAVE] button.   
+    The *Group saved* pop-up window is displayed.
+    
+    ![Group saved](../../Assets/Screenshots/Core1Platform/AdministratingCore1/GroupSaved.png "[Group saved]")
+
+    The newly created group is displayed in the list of groups.
+
+    ![Group created](../../Assets/Screenshots/Core1Platform/AdministratingCore1/GroupCreated.png "[Group created]")
+ 
+
+
+## Add user to user group
+
+Add any number of users to a user group to assign them rights subsequently.
+
+#### Prerequisites
+
+
+- A user group has been created, see [Create a user group](#create-a-user-group).
+
+#### Procedure
+
+*Settings > Users and groups > Tab GRUPPENVERWALTUNG (GROUP MANAGEMENT)*
+
+![Group management](../../Assets/Screenshots/Core1Platform/AdministratingCore1/GroupCreated.png "[Group management]")
+
+1. Select the checkbox of the user group for which you want to add or remove users.   
+    The editing toolbar is displayed.
+
+2. Click the ![Edit](../../Assets/Icons/Edit01.png "[Edit]") (Edit) button in the editing toolbar.       
+    The *Edit group "group name"* view is displayed. By default, the *Users* tab is preselected.
+
+    > [Info] If no users have been added yet, the *No users added. Use + button to add one to group* notice is displayed. Otherwise, the details of the user(s) included in the group are displayed.   
+    
+     ![Edit group no users](../../Assets/Screenshots/Core1Platform/AdministratingCore1/GroupEditNoUsers.png "[Edit group no users]")
+
+
+3. Click the ![Add](../../Assets/Icons/Plus01.png "[Add]") (Add) button in the bottom right corner.    
+    The *Add user* window is displayed.
+
+    ![Add user](../../Assets/Screenshots/Core1Platform/AdministratingCore1/GroupAddUser.png "[Add user]")
+
+4. Click the *Add user* drop-down list and select the desired user.   
+    The selected user has been added to the group. Per default, the validity period is set from today to the system end date. 
+
+    ![Add user](../../Assets/Screenshots/Core1Platform/AdministratingCore1/GroupAddUserValidity.png "[Add user]")
+
+     1. Click the ![Calendar](../../Assets/Icons/Calendar.png "[Add]") (Calendar) button and select the validity start date in the *Valid from* field.  
```

Comment:
I.d.R. haben wir immer solche Schritte als normaler Schritt beschrieben, nicht als "Unterschritt". An sich ist es auch vom "Add user" unabhängig. Evtl. könnte beide (Valid from und End date) in einem Schritt zusammentun.

Reply:
habe ich mir schon was bei gedacht